packages/integrations/cloudflare/src/index.ts

```diff
@@ -98,8 +98,18 @@ export type Options = {
 	 * See https://developers.cloudflare.com/kv/concepts/kv-namespaces/ for more details on using KV namespaces.
 	 *
 	 */
-
 	sessionKVBindingName?: string;
+
+	/**
+	 * The path to the entry file for the server. This is used to determine the entry point for the server.
+	 * By default, this is set to `@astrojs/cloudflare/entrypoints/server.js`.
+	 */
+	entryfilePath?: string;
+
+	/**
+	 * The exports to use for the server entry file. By default, this is set to `['default']`. If you need to have other top level exports, e.g. DurableObjects you need to provide them here.
+	 */
+	entryfileExports?: string[];
```

Comment:
Can we get some opinions on naming here please?
Also would it make sense to put them under a common prefix, e.g. `entryfile.path` & `entryfile.exports`

Reply:
Changed